classes/class-dibs-easy-gateway.php

```diff
@@ -291,6 +293,25 @@ public function process_dibs_payment_in_order( $order_id ) {
 		}
 	}
 
+	/**
+	 * Save shipping reference to Order.
+	 *
+	 * @param int $order_id order id.
+	 * @return void
+	 */
+	public function nets_save_shipping_reference_to_order( $order_id ) {
+		$packages        = WC()->shipping->get_packages();
+		$chosen_methods  = WC()->session->get( 'chosen_shipping_methods' );
+		$chosen_shipping = $chosen_methods[0];
+		foreach ( $packages as $i => $package ) {
+			foreach ( $package['rates'] as $method ) {
+				if ( $chosen_shipping === $method->id ) {
+					update_post_meta( $order_id, 'nets_shipping_reference', 'shipping|' . $method->id );
```

Comment:
Is it good for readability to store the data in a visual post meta? I mean like _nets_shipping_reference_ instead _of _ nets_shipping_reference_?

Reply:
If it is readable then its also editable by default from the merchants side. If the merchant changes this for any reason it might break the thing we are fixing with this. That is the only issue i can see. But it also lets the merchant change it if we ever get it wrong as well.